server/graphql/v2/interface/Account.js

```diff
@@ -300,20 +300,56 @@ const accountFieldsDefinition = () => ({
       ...CollectionArgs,
       onlyPublishedUpdates: { type: GraphQLBoolean },
       onlyChangelogUpdates: { type: GraphQLBoolean },
+      sortBy: { type: GraphQLString },
```

Comment:
In `ExpensesQuery`, `TransactionsQuery` and `OrdersQuery` it seems that we use `orderBy`, with the `ChronologicalOrderInput` type. Would be great to make it consistent. See https://github.com/opencollective/opencollective-api/blob/b1737a73c4c770a27eefd505749c45cb52d01e2d/server/graphql/v2/query/ExpensesQuery.ts#L105

Reply:
Thanks. Fixed. 👍🏽